api-docs/leads.md

```diff
@@ -0,0 +1,332 @@
+# Leads
+
+The leads document direct encounters with leads.
+
+Leads are potiental clients.
+
+Here is the list of the valid lead types:
```

Comment:
These look like clientLog types, not lead types. Right? The leads objects themselves don't have a `type`.